helm/cluster-autoscaler-app/values.yaml

```diff
@@ -8,17 +8,32 @@ groupID: 65534
 port: 8085
 
 # Make sure to update the docs at https://github.com/giantswarm/docs/blob/master/src/layouts/shortcodes/autoscaler_utilization_threshold.html if you change the defaults below
-configmap:
-  expander: "least-waste"
-  scanInterval: "10s"
-  scaleDownUtilizationThreshold: 0.65
-  skipNodesWithLocalStorage: "false"
-  skipNodesWithSystemPods: "true"
-
 image:
   registry: quay.io
   name: giantswarm/cluster-autoscaler
   tag: v1.15.2
 
-clusterID:
-  "test-cluster"
+# clusterID is dynamic environment value, calculated after cluster creation
+# applies only to Giant Swarm clusters
+clusterID: "test-cluster"
+# provider is dynamic environment value, which comes from application catalog configuration
+# applies only to Giant Swarm clusters
+provider: "changeme"
```

Comment:
Minor but in other apps we used a valid value like `aws` and then we override it. Should we do the same here?

https://github.com/giantswarm/nginx-ingress-controller-app/blob/963189b8e0981e65810e41a65d027c65f64e948b/helm/nginx-ingress-controller-app/values.yaml#L17

Reply:
ok, I have no hard opinions on it.